features/step_definitions/run_steps.rb

```diff
@@ -14,12 +14,14 @@
 
 
 When(/^I run `(.+?)` and enter "(.+?)"$/) do |command, input|
-  @result = run command, input: input
+  inputs = Kappamaki.from_sentence(input)
+  @result = run command, inputs: inputs
```

Comment:
It doesn't seem to work, but it also feels hackier. Why do it that way, when semantically we have multiple inputs?